examples/desktop/main.go

```diff
@@ -73,20 +73,15 @@ func main() {
 
 	openvpn3.SelfCheck(logger)
 
-	session := openvpn3.NewSession(&loggingCallbacks{})
-
 	bytes, err := ioutil.ReadFile(profileName)
```

Comment:
`profileName` comes from `os.Args[1]` and panics if now arguments were passed.

Reply:
Fixed